python_scripts/02_numerical_pipeline_hands_on.py

```diff
@@ -93,41 +96,55 @@
 data[numerical_columns].head()
 
 # %% [markdown]
-# Now that we limited the dataset to numerical columns only, we can check
-# closely what these numbers represent. Discarding `"fnlwgt"` aside for the
-# moment, we can identify two types of usage.
+# Now that we limited the dataset to numerical columns only,
+# we can analyse these numbers to figure out what they represent.
+# Discarding `"fnlwgt"` for the moment, we can identify two types of usage.
 #
-# The former is related to a measurement such as age. The data are continuous
-# meaning that they can take any value in a range. We can give the range for
-# age column as an example,
+# The first column, `"age"`, is self-explanatory.
+# We can note that the values are continuous, meaning they can take up any
+# number in a given range.
+# Let's find out what this range is:
 
 # %%
 data["age"].describe()
 
 # %% [markdown]
-# The age varies between 17 years and 90 years and can take any value in this
-# range.
+# We can see the age varies between 17 and 90 years.
 #
-# The latter is related categorical data. These data are discrete, in contrast
-# with continuous. It means that the variable can take only certain values
-# which are known as categories. We will come back later on this type of data
-# and how to handle them. Here, we are only interested to recognize them.
+# The next columns contain related categorical data.
+# These are discrete ; in contrast with continuous, they can only take a
+# value from a finite set, which we call categories.
+# We will come back later on this type of data and how to handle them.
+# Here, we are only interested in recognizing them.
 #
-# Here, the column `"education-num"` gives an example. The number encode the
-# education level which can only correspond to specific values. We can quickly
-# check the number of occurrence of each category to get convinced.
+# Let’s take for example “education-num”.
```

Comment:
Let's avoid quirky `"` (i.e. different one for opening and closing) character ...

Reply:
Nice catch, a quirk of my editor...

Reply:
```suggestion
# Let's take for example "education-num".
```